src/hw3/homer_downloader.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
+import urllib.request
+from bs4 import BeautifulSoup
+
+URL = "https://www.thisfuckeduphomerdoesnotexist.com/"
+
+
+def get_homer_image_url() -> str:
+    html = urllib.request.urlopen(URL).read()
+    image_url = BeautifulSoup(html, "html.parser").find(id="image-payload")["src"]
+    return image_url
+
```

Comment:
Are you sure that not defining this function as async one will not stop async event loop?